### DISCUSSION TITLE
有没有小伙伴能提供一下  kamailio IMS模块是 mmtel 模块的cfg 配置的

### DISCUSSION THREAD

**Discussion by Helloworld1668** [rts-cn/rts#21] [Kamailio]
Hi, 大家好
      目前在研究kamailio IMS 模块的代码，现在已经将P/I/S 网元起来了，并且模拟工具能打通 VoLte 电话了，我想问问老哥们，有没有mmetl 网元的cfg 配置，不胜感激！   不胜感激，必有重谢！

**Comment by Helloworld1668** [comment]
杜老师
       我是看你书入得门，kamailio 实战这本书太棒了，给了我非常大的帮助，再次先万分感谢！

**Reply by seven1240** [reply]
很高兴能帮上你 :)

**Comment by Helloworld1668** [comment]
你好，杜老師，我是湘潭大学的学生，现在在实验室里做5GC  VoNR 的项目，用的是kamailio的开源项目，目前P/I/S-CSCF 能跑起来，也能用模拟器打通基本语音业务；现在还差AS ，回铃音等模块；这个as  mmtel 的cfg 配置有什么资料可以提供一下吗，希望得到你的指导帮助。当然我是有偿的，实验室有研发经费。

2024-01-02 17:47:01 "Seven Du" ***@***.***> 写道：

很高兴能帮上你 :)
